airflow/models/dagrun.py

```diff
@@ -1411,7 +1411,12 @@ class DagRunNote(Base):
     __tablename__ = "dag_run_note"
 
     user_id = Column(Integer, nullable=True)
-    dag_run_id = Column(Integer, primary_key=True, nullable=False)
+    dag_run_id = Column(
+        Integer,
+        primary_key=True,
+        nullable=False,
+        foreign_key=ForeignKey("ab_user.id", name="dag_run_note_user_fkey"),
+    )
```

Comment:
Shouldn’t the fk be on the `user_id` column instead?

Reply:
My bad, yes I'll fix it